### PR TITLE
[Feature] Spotify Template

### PR DESCRIPTION
**Pull request by Manish-Marvel**

### CODE REVIEW

**Comment by Manish-Marvel** [comment]
@sparrow007 Please Review

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -8,30 +8,44 @@
...
+            android:label="@string/title_spotify_screen"
+            android:screenOrientation="portrait"
+            android:theme="@style/Theme.FUITemplet" >
+            <intent-filter>
+                <action android:name="android.intent.action.MAIN" />
+
+                <category android:name="android.intent.category.LAUNCHER" />
+            </intent-filter>
+
+            <meta-data
```
Comment (sparrow007) [inline app/src/main/AndroidManifest.xml]: Please remove this

File: app/src/main/java/com/marvel999/fuitemplet/Spotify/SpotifyActivity.kt [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+            modifier = Modifier
+                .wrapContentSize()
+                .align(Alignment.Center)
+        ) {
+            Image(
+                modifier = Modifier
+                    .size(100.dp)
+                    .align(Alignment.CenterHorizontally),
+                painter = painterResource(id = R.drawable.ic_spotify_logo),
+                contentDescription = "spotify Splash screen"
```
Comment (sparrow007) [inline app/src/main/java/com/marvel999/fuitemplet/Spotify/SpotifyActivity.kt]: Plesase put these strings in strings.xml

File: app/src/main/java/com/marvel999/fuitemplet/Spotify/SpotifyHomeActivity.kt [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+                    )
+                },
+                selectedContentColor = Color.White,
+                unselectedContentColor = offWhiteTextColor,
+                alwaysShowLabel = true,
+                selected = currentRoute == item.screen_route,
+                onClick = {
+                    navController.navigate(item.screen_route) {
+
+                        navController.graph.startDestinationRoute?.let { screen_route ->
```
Comment (sparrow007) [inline app/src/main/java/com/marvel999/fuitemplet/Spotify/SpotifyHomeActivity.kt:96]: Please check if we can remove the navcontroller?

File: app/src/main/java/com/marvel999/fuitemplet/Spotify/component/RowMusicItem.kt [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+                    modifier = Modifier
+                        .padding(start = 16.dp, bottom = 20.dp)
+                        .align(Alignment.BottomStart),
+                    color = tertiary,
+                    maxLines = 1,
+                    overflow = TextOverflow.Ellipsis,
+                    style = Typography.titleLarge,
+                    fontSize = 22.sp
+                )
+                val color:Color = Color(parseColor(randomHexColorGanrator()))
```
Comment (sparrow007) [inline app/src/main/java/com/marvel999/fuitemplet/Spotify/component/RowMusicItem.kt]: Please remove explicit data type

File: app/src/main/java/com/marvel999/fuitemplet/Spotify/screen/SpotifyHome.kt [hunk 5]
```diff
@@ -0,0 +1,217 @@
...
+            .fillMaxSize()
+            .background(dark)
+            .padding(start = 16.dp, bottom = 60.dp)
+    ) {
+        item {
+            SpotifyHeader(modifier = Modifier.padding(top = 15.dp))
+        }
+        stickyHeader {
+            RowTopic()
+            Spacer(modifier = Modifier.padding(top = 20.dp))
```
Comment (sparrow007) [inline app/src/main/java/com/marvel999/fuitemplet/Spotify/screen/SpotifyHome.kt]: Use height instead of padding

**Review by sparrow007 — CHANGES_REQUESTED** [review]
Please check all the comments

File: app/src/main/AndroidManifest.xml (hunk 1 above)
Comment (Marvel999) [reply app/src/main/AndroidManifest.xml]: done

File: app/src/main/java/com/marvel999/fuitemplet/Spotify/SpotifyActivity.kt (hunk 2 above)
Comment (Marvel999) [reply app/src/main/java/com/marvel999/fuitemplet/Spotify/SpotifyActivity.kt]: done

File: app/src/main/java/com/marvel999/fuitemplet/Spotify/SpotifyHomeActivity.kt (hunk 3 above)
Comment (Marvel999) [reply app/src/main/java/com/marvel999/fuitemplet/Spotify/SpotifyHomeActivity.kt:96]: it navoptionBuilder scope not navController

File: app/src/main/java/com/marvel999/fuitemplet/Spotify/component/RowMusicItem.kt (hunk 4 above)
Comment (Marvel999) [reply app/src/main/java/com/marvel999/fuitemplet/Spotify/component/RowMusicItem.kt]: done

File: app/src/main/java/com/marvel999/fuitemplet/Spotify/screen/SpotifyHome.kt (hunk 5 above)
Comment (Marvel999) [reply app/src/main/java/com/marvel999/fuitemplet/Spotify/screen/SpotifyHome.kt]: done

**Review by Marvel999 — APPROVED** [review]

**Pull request merged**